pshtt/pshtt.py

```diff
@@ -16,6 +16,7 @@
 import pytablewriter
 import sys
 import codecs
+import OpenSSL
```

Comment:
I'm getting `ImportError: No module named 'OpenSSL'` because of the `import OpenSSL` line. Did you install any additional dependencies not captured in `requirements.txt`?